src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/FancyZonesViewModel.cs

```diff
@@ -378,16 +389,16 @@ public Color ZoneBorderColor
 
             set
             {
-                if (value != _zoneBorderColor)
+                if (!value.Equals(_zoneBorderColor))
```

Comment:
Equals should have a StringComparison.OrdinalIgnoreCase passed into it at the very least

Reply:
updated here: https://github.com/microsoft/PowerToys/pull/5762/commits/b8f95055b0a5ec67adb1e37c577ef08100151a3b